custom_components/aquarea/definitions.py

```diff
@@ -557,9 +557,7 @@ def guess_shift_or_direct_and_clamp_min_max_values(
     HeishaMonBinarySensorEntityDescription(
         heishamon_topic_id="TOP93",
         key="panasonic_heat_pump/main/Pump_Duty",
-        name="Aquarea Pump Running",
-        state=bit_to_bool,
-        device_class=BinarySensorDeviceClass.RUNNING,
```

Comment:
I agree with entity renaming but don't really understand removal of device_class and state conversion, could you clarify why it is necessary?

Reply:
Oh, yes it should go to  HeishaMonSensorEntityDescription.. I think I will leave it as was before and add additional in Sensors.. Quick fix :)